Packs/PAN-OS/Integrations/Panorama/Panorama.yml

```diff
@@ -4564,7 +4564,59 @@ script:
     - contextPath: Panorama.UserIDAgents.Disabled
       description: Whether the user-ID agent is disbaled.
       type: String
-  dockerimage: demisto/python3:3.9.2.17246
+  - name: panorama-upload-content-update-file
+    arguments:
+    - name: entryID
+      required: true
+      description: Entry ID of the file to upload.
+    - name: category
+      required: true
+      auto: PREDEFINED
+      predefined:
+      - wildfire
+      - anti-virus
+      - content
+      description: The file type
```

Comment:
Is this the file type or the category of the file?

Reply:
@apekarovsky Please see that comment

Reply:
Checked with @apekarovsky , should be a category of the content.
Changed.